docs/kotlin-sdk/integrate-with-okta.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Integrate With Okta
+sidebar_position: 2
+---
+
+**[ROUGH DRAFT, needs review for accuracy and editing]**
+
+This guide describes how to configure Beyond Identity with an use an OIDC authentication flow without a management API resource server. 
+This flow should be used if a client is using Beyond Identity for authentication only.
+
+### Prerequisites
+
+ - [Integrate With Okta](https://developer-docs-git-v1-beyondidentity.vercel.app/docs/v1/sso-integrations/integrate-with-okta)
```

Comment:
We should use relative paths here, just so it works in any deployed environment.

Reply:
Updated